src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java

```diff
@@ -559,4 +564,16 @@ public VisualEditorAddMediaDialog searchImage(String searchInput) {
     return mediaDialog;
   }
 
+  public VisualEditorPageObject  getTextArea() {
```

Comment:
Strange spacing.
This method has a very misleading name, as it does not return a WebElement - it actually performs a click action on the text area. In this case, let's either rename it to clickTextArea() or return a webElement from it and perform a click action in the test body.